addons/im_livechat/static/src/views/discuss_template.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+    <t t-name="im_livechat.LivechatDiscuss">
+        <Discuss hasSidebar="false" thread="thread"/>
```

Comment:
We might want to be careful here, if `get_channel` returns no thread, the props will be falsy, and the getter in discuss will display `this.store.discuss.thread` rather than no thread, which is not what we want.